src/field_move.c

```diff
@@ -161,7 +161,7 @@ void Fsys_MakeFieldMoveCheckData(FieldSystem *fsys, struct FieldMoveCheckData *c
     if (MetatileBehavior_IsWhirlpool(facingTile)) {
         checkData->flag |= (1 << FIELD_MOVE_CHECK_WHIRLPOOL_F);
     }
-    if (FlyPoints_GetWeatherType(Save_FlyPoints_Get(fsys->savedata)) == 11) {
+    if (LocalFieldData_GetWeatherType(Save_LocalFieldData_Get(fsys->savedata)) == 11) {
```

Comment:
I forget, do we have constants for the Weather Types in the overworld yet?

Reply:
as far as I know, we don't